enhancements/storage/cleanup-lso-symlink.md

```diff
@@ -0,0 +1,66 @@
+---
+title: handle-local-volume-deletion
+authors:
+  - "@gnufied"
+reviewers:
+  - "@jsafrane”
+  - "@fbertina"
+  - "@chuffman"
+approvers:
+  - "@..."
+creation-date: 2021-04-17
+last-updated: 2021-04-17
+status: implementable
+see-also: https://github.com/openshift/enhancements/blob/master/enhancements/storage/cleanup-lso-symlink.md
+replaces:
+superseded-by:
+---
+
+# Cleanup and handling of LocalVolume and LocalVolumeSet deletion
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a mechanism that will allow PVs provisioned by local-storage-operator(LSO) to be deleted and corresponding symlinks to be removed from the node when `LocalVolume` or `LocalVolumeSet` objects are deleted.
+
+## Motivation
+
+Currently when a user deletes `LocalVolume` or `LocalVolumeSet` objects - we do not automatically delete the PVs created from those objects and neither do we remove the symlinks created on the node. This has been documented as a limitation because removal of symlinks and PVs can be sometimes race-prone.
+
+This design fixes that problem and ensures that both PVs and symlinks are removed when `LocalVolume` and `LocalVolumeSet` objects are removed.
+
+### Goals
+
+* Do not allow deletion of `LocalVolumeSet` object if it has bound PVCs.
```

Comment:
Seeing it written like this: why do we allow deletion of LocalVolumes then? Please add a note that LocalVolumes can't be deleted already.

Reply:
done.